source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Tests/Application/Validation/InputValidation/ValidationRules/ChargeTypeIsKnownValidationRuleTest.cs

```diff
@@ -29,6 +29,7 @@ public class ChargeTypeIsKnownValidationRuleTest
         [InlineAutoMoqData(ChargeType.Fee, true)]
         [InlineAutoMoqData(ChargeType.Subscription, true)]
         [InlineAutoMoqData(ChargeType.Subscription, true)]
```

Comment:
Subscription was mentioned twice
```suggestion
        [InlineAutoMoqData(ChargeType.Tariff, true)]
```

Reply:
done!